python_transport/wirepas_gateway/dbus/sink_manager.py

```diff
@@ -92,6 +102,14 @@ def _on_stack_started(self, sender, object, iface, signal, params):
 
         self.on_stack_started(self.sink_id)
 
+    def _on_stack_stopped(self, sender, object, iface, signal, params):
+        # pylint: disable=unused-argument
+        # pylint: disable=redefined-builtin
+        # Force update of network address in case remote api modify it
+        self.get_network_address(True)
```

Comment:
It is not needed.
It is done in start because of remote api reboot, but on stop, their is no need for it. Our last value read from start is for sure valid.

Reply:
done